WIP-CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 
 - The sc-logs-retention cronjob now runs without error even if no backups were found for automatic removal
 - Harbor Swift authentication configuration options has moved from `citycloud` to `harbor.persistence.swift`.
+- The dry-run and apply command now checks against the state of the cluster while ran.
```

Comment:
Update this so that it reads as optional now since the functionality is behind optional flags.